adr/2024-12-19-offer-html-alternative-to-our-pdf-standard-document.md

```diff
@@ -0,0 +1,169 @@
+---
+title: [A11y] Offer HTML alternative to our pdf standard documents
+date: 2024-12-19
+area: accessibility
+tags: [a11y, pdf, html]
+---
+
+## Context
+
+To comply with Web Content Accessibility Guidelines (WCAG), we aim to make Shopware 6's document solution accessible (A11y-compliant). This ensures that our solution remains usable while meeting legal requirements and making documents accessible to customers with disabilities. 
+
+Currently, our PDF generation library, DomPDF, does not meet accessibility standards, posing a significant challenge.
+
+## Decision
+
+We have decided to switch from tagged PDFs to HTML for better accessibility.
```

Comment:
I wouldn't call it `switch`, as that sounds like we are replacing PDFs. Also we don't have tagged PDFs currently.
```suggestion
We have decided to make HTML documents available in addition to PDF documents, as these are more accessible.
```